src/Command/Analyze.php

```diff
@@ -131,6 +131,7 @@ private function addLogLevelFormatsTo(OutputInterface $output)
     private function createConfigBy(InputInterface $input)
     {
         $this->configFilePath = realpath($input->getArgument('config'));
+        chdir(dirname($this->configFilePath));
```

Comment:
I like the idea, but i think this leads to sideeffects. 
Changing the working directory has influence to the entire tool.
A better approach would be to solve the path by something like this..

´´´php
function generateAbsolutePathFrom($path) {
    if ($path[0] === '/' || strpos($path, '://') !== false) {
        return $path;
    }

    return dirname($this->configFilePath) . DIRECTORY_SEPARATOR . $path;
}
´´´